<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainw</class>
 <widget class="QMainWindow" name="mainw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1482</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainw</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="gw_xy">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>1</y>
      <width>1330</width>
      <height>460</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>UI</string>
    </property>
    <widget class="QPushButton" name="pb_home">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_led1">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_pause">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_resume">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_led3">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_led2">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>480</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="nm_stepx">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>25</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepy">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepz">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>65</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepa">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>85</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>26</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>46</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>65</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>85</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>A:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>3</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Step Coords</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepy_2">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>45</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepa_2">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>85</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepz_2">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>65</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_stepx_2">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>25</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>3</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>pwr</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_end">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_const">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Const</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_start">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>accel:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_acc">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Acc</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_dec">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dec</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="tx_commands">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>531</width>
      <height>131</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_feed_line">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FeedLine</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1334</x>
      <y>0</y>
      <width>101</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="nm_real_a">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QDial" name="dl_rotor_z">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchesVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_rotor_y">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_rotor_z">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="nm_real_y">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QDial" name="dl_rotor_y">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchesVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="nm_real_z">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_rotor_x">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="nm_rotor_a">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="nm_real_x">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QDial" name="dl_rotor_a">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchesVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDial" name="dl_rotor_x">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchesVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_bloc_execution">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Block exec</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="vs_simu_speed">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>490</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
